src/apscheduler/datastores/memory.py

```diff
@@ -207,8 +207,11 @@ async def acquire_schedules(self, scheduler_id: str, limit: int) -> list[Schedul
         schedules: list[Schedule] = []
         for state in self._schedules:
             if state.next_fire_time is None or state.next_fire_time > now:
-                # The schedule is either paused or not yet due
-                break
+                # The schedule is either exhausted or not yet due
+                continue
```

Comment:
Ah right, I was mistakenly under the impression that other schedules past this point could be due. I've reverted this to use `break` instead of `continue`, and left a clarifying comment to hopefully help other contributors in the future not make my mistake.